src/SVGSkin.js

```diff
@@ -94,18 +103,26 @@ class SVGSkin extends Skin {
      * @return {SVGMIP} An object that handles creating and updating SVG textures.
      */
     createMIP (scale) {
-        this._svgRenderer.draw(scale);
+        const {width, height} = this._svgMeasurements;
+        this._canvas.width = width * scale;
+        this._canvas.height = height * scale;
+        // Even if the canvas at the current scale has a nonzero size, the image's dimensions are floored pre-scaling.
+        // e.g. if an image has a width of 0.4 and is being rendered at 3x scale, the canvas will have a width of 1, but
+        // the image's width will be rounded down to 0 on some browsers (Firefox) prior to being drawn at that scale.
```

Comment:
note to self: move this comment in the middle of the condition so it's clear it refers specifically to the naturalWidth/naturalHeight check